code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -110,6 +172,18 @@
     assembled = 2
     active = 0
 
+/obj/item/weapon/grenade/iedcasing/preassembled/withshrapnel
+//	var/list/shrapnel_list = new()
+	var/please_work = "obj/item/toy/crayon/red"
```

Comment:
I forgettied to remove this. When trying out early versions of the code I had it preloaded with red crayons. This particular line I wrote while trying to figure out how to even load objects into lists.